examples/ts-react-chat/src/routes/generations.audio.tsx

```
@@ -0,0 +1,359 @@
+import { useEffect, useMemo, useRef, useState } from 'react'
+import { createFileRoute } from '@tanstack/react-router'
+import { useGenerateAudio } from '@tanstack/ai-react'
+import type { UseGenerateAudioReturn } from '@tanstack/ai-react'
```

ESLint

[error] `@tanstack/ai-react` type import should occur after import of `../lib/audio-providers`

(import/order)